src/dymoprint/dymo_print_engines.py

```diff
@@ -119,6 +119,96 @@ def render_barcode(
         )
         return code_bitmap
 
+    def __create_custom_code_bitmap(self, barcode_input_text, bar_code_type):
+        """
+        Renders a barcode image from the input text and barcode type.
+
+        Args:
+            barcode_input_text (str): The input text to be encoded in the barcode.
+            bar_code_type (str): The type of barcode to be rendered.
+
+        Returns:
+            Image: A barcode image.
+        """
+        label_height = DymoLabeler.max_bytes_per_line(self.tape_size_mm) * 8
+        if len(barcode_input_text) == 0:
+            return Image.new("1", (1, label_height))
+
+        code = barcode_module.get(
+            bar_code_type, barcode_input_text, writer=BarcodeImageWriter()
+        )
+
+        code_bitmap = code.render(
+            {
+                "font_size": 10,
+                "vertical_margin": 2,
+                "module_height": (DymoLabeler.max_bytes_per_line(self.tape_size_mm) * 6)
+                - 16,
+                "module_width": 1,
+                "background": "black",
+                "foreground": "white",
+            }
+        )
+        return code_bitmap
+
+    def render_barcode_with_text(
+        self,
+        barcode_input_text,
+        bar_code_type,
+        font_file_name: str,
+        frame_width,
+        font_size_ratio=0.9,
+        align="center",
```

Comment:
I'm curious: is there a use-case for non-centered barcode text? I've only seen centered text with a barcode.

The reason I'm asking is that we might want to simply hard-code the alignment to be center in order to reduce code complexity.

Reply:
If it works and is not a mandatory parameter then I would leave it as is. I can think of a scenario when left side alignment could be desirable.